_posts/2017-11-03-nearing-the-7.0-release.md

```diff
@@ -64,46 +65,46 @@ We will also be working on a upgrade tool that will help [rewrite your package.j
+Even though the amount of work that goes into maintaining the lists of data is humongous — again, why we need help — it solves multiple issues. It makes sure users are up to date with the spec. It means less configuration/package confusion. It means an easier upgrade path. And it means less issues about what is what.
 
 `babel-preset` is actually a pretty *old* preset that replaces every other syntax preset that you will need (es2015, es2016, es2017, es20xx, latest, etc...).
 
 ![npm presets](https://i.imgur.com/nNKKFcp.png)
 
-It compiles the latest yearly release of JavaScript (whatever is in Stage 4) which replaces all the old presets. But it also has the further abilitiy to compile according to the target environments you specify: whether it's for development mode (latest version of a browser), or different builds (one for IE, one for evergreen browsers).
+It compiles the latest yearly release of JavaScript (whatever is in Stage 4) which replaces all the old presets. But it also has the abilitiy to compile according to target environments you specify: whether that is for development mode, like the latest version of a browser, or for multiple builds, like a version for IE, and then another version for evergreen browsers.
 
 ### Not removing the Stage presets (babel-preset-stage-x)
 
 <blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Frustration level if we remove the Stage presets in Babel? (in favor explicitly requiring proposal plugins since they aren&#39;t JavaScript yet)</p>&mdash; Henry Zhu (@left_pad) <a href="https://twitter.com/left_pad/status/873242704364306433?ref_src=twsrc%5Etfw">June 9, 2017</a></blockquote>
 
 We can always keep it up to date, and maybe we just need to determine a better system than what these presets are currently.
 
-Right now, they are literally just a list of plugins that we manually update after a TC39 meeting happens. Of course to make this managable we will need to just allow for major version bumps for these "unstable" packages. Part of the reason for this is because the community will just re-create these packages anyway, so we might as well do it via an official package and have the ability to provide better messaging, etc.
+Right now, stage presets are literally just a list of plugins that we manually update after TC39 meetings. To make this managable, we need to allow major version bumps for these "unstable" packages. Part of the reason for this is because the community will re-create these packages anyway, so we might as well do it from an official package and then have the ability to provide better messaging, etc.
 
 ### Renames: Scoped Packages (`@babel/x`)
 
-Alright we only just changed every single package for Babel again 😂..
+Alright, we only just changed every single package for Babel again 😂..
```

Comment:
I would prefer to not include this sentence at all. We don't need to say that "we broke everything for you once again".

Reply:
It's not broken though? renaming a package doesn't break the existing package, but sure we can remove it if you want

Reply:
No it's not but I mean that's what I understand when I read this.